index.html

```diff
@@ -4755,15 +4755,19 @@ <h2>Definition of Roles</h2>
           <rdef>group</rdef>
           <div class="role-description">
             <p>A set of user interface <a>objects</a> that is not intended to be included in a page summary or table of contents by <a>assistive technologies</a>.</p>
-            <p>Contrast with <rref>region</rref>, which is a grouping of user interface objects that will be included in a page summary or table of contents.</p>
             <p>
-              Authors SHOULD use a <code>group</code> to form a logical collection of items in a <a>widget</a>, such as children in a tree widget forming a collection of siblings in a hierarchy.
-              However, when a <code>group</code> is used in the context of a <rref>listbox</rref>, for example, authors MUST limit its children to <rref>option</rref> elements. Therefore, proper
-              handling of <code>group</code> by authors and assistive technologies is determined by the context in which it is provided.
-            </p>
+              Contrast with <rref>region</rref>, which is a grouping of user interface objects that will be included in a page summary or table of contents. If a section is significant enough to 
+	      warrant inclusion in the web page's table of contents, the author SHOULD assign it a <a>role</a> of<rref>region</rref> or a <a href="#landmark_roles">standard landmark role</a>.
+	    </p>
             <p>
-              Authors MAY nest <code>group</code> elements. If a section is significant enough to warrant inclusion in the web page's table of contents, the author SHOULD assign it a <a>role</a> of
-              <rref>region</rref> or a <a href="#landmark_roles">standard landmark role</a>.
+              Authors SHOULD use a <code>group</code> to form a logical collection of items in a <a>widget</a>, such as children in a tree widget forming a collection of siblings in a hierarchy.
+	      Authors MAY nest <code>group</code> elements.
+	    </p>
+	    <p>
+	      The proper handling of <code>group</code> by authors and assistive technologies is determined by the context in which it is provided. For example, when a <code>group</code> is the 
+	      <a>accessibility child</a> of a <code>menu</code> or <code>menubar</code>, authors MAY nest additional <code>group</code> elements. However, the <a>accessibility children</a> of the 
```

Comment:
I think the new content (However, ...) is  good addition but  it should be a note and not using normative language. As my suggestion for tree widgets shows, it's easy for such language to be out of date.  I think normative statements on roles should be limited to their own sections.